src/Adapters/Storage/Mongo/MongoStorageAdapter.js

```diff
@@ -187,6 +188,52 @@ export class MongoStorageAdapter implements StorageAdapter {
     }
     return this.client.close(false);
   }
+  async getSecurityLogs(req: any) {
+    const options = req.config || req;
+    let databaseURI = options.databaseURI;
+    const warnings = [];
+    if (databaseURI.includes('@')) {
+      databaseURI = `mongodb://${databaseURI.split('@')[1]}`;
+      const pwd = options.databaseURI.split('//')[1].split('@')[0].split(':')[1] || '';
+      if (!pwd.match('^(?=.*[a-z])(?=.*[A-Z])(?=.*[0-9])(?=.*[!@#$%^&*])(?=.{14,})')) {
+        warnings.push({
+          title: `Weak Database Password`,
+          message:
+            'The database password set lacks complexity and length. This could potentially allow an attacker to brute force their way into the database, exposing the database.',
+        });
+      }
+    }
+    let databaseAdmin = '' + databaseURI;
+    try {
+      const parsedURI = url.parse(databaseAdmin);
+      parsedURI.port = '27017';
+      databaseAdmin = parsedURI.toString();
+    } catch (e) {
+      /* */
+    }
```

Comment:
Only raise this db authentication warning in NODE_ENV production

Reply:
I don't see any harm logging this in development? Enabling checks depending on the environment is something I am critical of. We deny the developer to test out security checks in a safe playground. This can create a whole lot of confusion and false sense of security, especially since the PR currently does not log which checks are actually conducted.